### PR TITLE
feat(cli): show whether a package is auto-updated (#296)

### LINKED ISSUE #296: Show whether a package is auto-updated
**Is your feature request related to a problem? Please describe.**
`glasskube describe` command and the package detail page do not show yet whether a package is marked for auto updates.

**Describe the solution you'd like**
Show whether auto updates are enabled at a package or not. This should be done for both CLI (ls and describe command) and UI (package detail page).

Packages are marked for auto updates if they have a label `packages.glasskube.dev/auto-update` with value `true`.

**Describe alternatives you've considered**

**Additional context**
Ref #338 #341

### PR DESCRIPTION
**Pull request by Baalekshan**
Closes #296 

## 📑 Description

This PR introduces the Auto-Update section in `list` and `describe` command which shows whether the package is marked for auto-updates.

```
$ ./glasskube ls
NAME                  STATUS         VERSION    AUTO-UPDATE
argo-cd               Not installed                      
cert-manager          Ready          v1.14.2+1  Enabled  
cyclops               Not installed                      
ingress-nginx         Not installed                      
keptn                 Ready          v0.10.0+1  Enabled  
kubernetes-dashboard  Not installed        
```
**If a package is installed** 
```
 $ ./glasskube describe keptn
Package: keptn — Toolkit for cloud-native application lifecycle management
Version: v0.10.0+1
Status:  Ready
Auto-Update: Enabled

References: 
 * Glasskube Package Manifest: https://packages.dl.glasskube.dev/packages/keptn/v0.10.0+1/package.yaml

Long Description:
Supercharge your deployments with Keptn! Keptn provides a “cloud-native” approach for managing the application release lifecycle metrics, observability, health checks, with pre- and post-deployment evaluations and tasks.
```

**If a package is not installed**

```
$ ./glasskube describe cyclops
Package: cyclops — Developer friendly Kubernetes
Version: v0.0.1+1
Status:  Not installed

Entrypoints:
 * Name: ui, Remote: cyclops-ui:3000, Local: http://localhost:3000/

References: 
 * Glasskube Package Manifest: https://packages.dl.glasskube.dev/packages/cyclops/v0.0.1+1/package.yaml
 * Github: https://github.com/cyclops-ui/cyclops
 * Website: https://cyclops-ui.com/

Long Description:
Welcome to Cyclops, a powerful user interface for managing and interacting with Kubernetes clusters.
Cyclops is designed to simplify the management of containerized applications on Kubernetes,
providing an intuitive and user-friendly experience for developers, system administrators, and DevOps professionals.
Divide the responsibility between your infrastructure and your developer teams so everyone can play to their strengths.
Automate your processes and shrink the window for deployment mistakes.
```

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

File: cmd/glasskube/cmd/describe.go [hunk 1]
```diff
@@ -35,6 +35,9 @@ var describeCmd = &cobra.Command{
 		fmt.Println(bold("Package:"), nameAndDescription(manifest))
 		fmt.Println(bold("Version:"), version(pkg, latestVersion))
 		fmt.Println(bold("Status: "), status(pkgStatus))
+		if pkg != nil {
+			fmt.Println(bold("Auto-Update:"), autoUpdateString(pkg))
```
Comment (christophenne) [inline cmd/glasskube/cmd/describe.go:39]: When auto updates are disabled, this just shows "Auto-Update: ", but I think in the describe command it would be better to show "Disabled". This is just my opinion, but I think it makes sense that in the list command we should only show "Enabled" but no "Disabled" (that way you can see the enabled ones faster), while in the describe command we show "Disabled". 

You could have an argument `disabledStr string` in the `autoUpdateString` function, such that the caller can define what should be shown when auto updates are disabled.

**Review by christophenne — CHANGES_REQUESTED** [review]
Thanks, good work so far. Please have a look at my comment.

File: pkg/install/install.go [hunk 2]
```diff
@@ -63,6 +63,10 @@ func (obj *installer) install(
 		pkg.SetLabels(map[string]string{
 			"packages.glasskube.dev/auto-update": "true",
 		})
+	} else {
```
Comment (christophenne) [inline pkg/install/install.go:66]: Is this change really necessary? I think everything should work without it too, right? Or was there some problem related to that?

**Review by christophenne — COMMENTED** [review]
Thanks for the update! Please have a look at the question and if appropriate, update the code again. 

Can you do us another favor please: We like to squash our commits before merging them into main, such that we don't have multiple commits with the same message but only one. Please read this: [Squashing](https://www.baeldung.com/ops/git-squash-commits). Afterwards you need to force-push your branch, and it will show up in the PR as one commit. While doing that, please also improve the commit message to something like "feat(cli): show whether a package is auto-updated (#296)", because it is not really a notifier. 
We didn't require this for first-time or newer contributors yet because we can also do it for them, but since you open PRs quite frequently, we kindly ask you to try it – that would help us a lot :)

File: pkg/install/install.go (hunk 2 above)
Comment (Baalekshan) [reply pkg/install/install.go:66]: I thought of removing it later but I forgot it was accidental

**Comment by Baalekshan** [comment]
@christophenne I will do this. Just a suggestion this could be added to contributor guidelines.

**Comment by christophenne** [comment]
> @christophenne I will do this. Just a suggestion this could be added to contributor guidelines.

Absolutely.

**Comment by Baalekshan** [comment]
FYI @christophenne , changes have been done!

**Review by christophenne — APPROVED** [review]
Great job, thank you!
